### ISSUE TITLE
bug: Weird audio glitch when zooming in and out in WASM build frantically

### ISSUE THREAD

**Issue by PraxTube** [PraxTube/lost-oppai#7]
I get

```
WARN /home/runner/.cargo/registry/src/index.crates.io-6f17d22bba15001f/bevy_text-0.12.1/src/glyph_brush.rs:120 warning[B0005]: Number of font atlases has exceeded the maximum of 16. Performance and memory usage may suffer.
```

when change the zoom level of the whole itch page really frantically (may also work when doing it slowly). I am already aware of UI glitches when zooming in and out but now the audio also seems to be effected.

**Comment by PraxTube** [comment]
So the UI glitches seemed to have been fixed by aef7b3e3d514f2ba6585adf2927e0f8e69a51965 by using

```
fit_canvas_to_parent: false,
```

though I don't know about the audio glitch.

**Comment by PraxTube** [comment]
I didn't encounter this ever since and I can also not reproduce this. I also didn't see anything like this during the 30+min playtest.
